src/walk.rs

```diff
@@ -82,6 +83,27 @@ pub fn scan(path_vec: &[PathBuf], pattern: Arc<Regex>, config: Arc<Options>) ->
         walker.add_custom_ignore_filename(".fdignore");
     }
 
+    // Global fdignore, has lowest precedence.
+    if let Some(global_ignore_file) = env::var_os("XDG_CONFIG_HOME")
+        .and_then(|x| if x.is_empty() { None } else { Some(PathBuf::from(x)) })
+        // std::env::home_dir() is deprecated. We should switch to the `dirs` crate in the future.
+        .or_else(|| env::home_dir().map(|p| p.join(".config")))
+        .map(|x| x.join("fd/ignore"))
```

Comment:
I guess it works on Windows as well, but maybe just use `.join("fd").join("ignore")`.

Reply:
Got it 👍.